pywren/serialize/cloudpickle/cloudpickle.py

```diff
@@ -12,7 +12,7 @@
 <https://web.archive.org/web/20140626004012/http://www.picloud.com/>`_.
 
```

Comment:
I think that's doable? @shivaram 
Let's then ignore those files, revert the pylint changes and merge this PR.